### ISSUE TITLE
Installlation failed

### ISSUE THREAD

**Issue by tastyminerals** [oh-my-fish/oh-my-fish#560]
**omf** fails to install, details below
```bash
fish install --path=~/.local/share/omf --config=~/.config/omf
Existing installation detected at /home/pavel/.local/share/omf
Would you like to remove the existing installation? (y/N): ⏎                                                                                                                                                       read> y
This will uninstall Oh My Fish and all plugins and themes from 
/home/pavel/.local/share/omf.
Are you sure you want to continue? (y/N): <E> fish: src/reader.cpp:1559: failed assertion: input_initialized
<E> fish: Backtrace:
<E> fish: 0   reader_push(wchar_t const*) + 1964
<E> fish: 1   fish() [0x436ac5]
<E> fish: 2   builtin_run(parser_t&, wchar_t const* const*, io_streams_t&) + 256
<E> fish: 3   exec_job(parser_t&, job_t*) + 1427
<E> fish: 4   parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1230
<E> fish: 5   parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 174
<E> fish: 6   parse_execution_context_t::eval_node_at_offset(unsigned int, block_t const*, io_chain_t const&) + 1680
<E> fish: 7   parser_t::eval_block_node(unsigned int, io_chain_t const&, block_type_t) + 242
<E> fish: 8   parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t, parse_node_tree_t) + 533
<E> fish: 9   parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t) + 477
<E> fish: 10  fish() [0x4721dc]
<E> fish: 11  exec_job(parser_t&, job_t*) + 6664
<E> fish: 12  parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1230
<E> fish: 13  parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 174
<E> fish: 14  parse_execution_context_t::eval_node_at_offset(unsigned int, block_t const*, io_chain_t const&) + 1680
<E> fish: 15  parser_t::eval_block_node(unsigned int, io_chain_t const&, block_type_t) + 242
<E> fish: 16  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t, parse_node_tree_t) + 533
<E> fish: 17  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t) + 477
<E> fish: 18  fish() [0x4721dc]
<E> fish: 19  exec_job(parser_t&, job_t*) + 6664
<E> fish: 20  parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1230
<E> fish: 21  parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 174
<E> fish: 22  parse_execution_context_t::eval_node_at_offset(unsigned int, block_t const*, io_chain_t const&) + 1680
<E> fish: 23  parser_t::eval_block_node(unsigned int, io_chain_t const&, block_type_t) + 242
<E> fish: 24  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t, parse_node_tree_t) + 533
<E> fish: 25  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t) + 477
<E> fish: 26  fish() [0x4721dc]
<E> fish: 27  exec_job(parser_t&, job_t*) + 6664
<E> fish: 28  parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1230
<E> fish: 29  parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 174
<E> fish: 30  parse_execution_context_t::run_if_statement(parse_node_t const&) + 426
<E> fish: 31  parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1267
<E> fish: 32  parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 174
<E> fish: 33  parse_execution_context_t::run_if_statement(parse_node_t const&) + 426
<E> fish: 34  parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1267
<E> fish: 35  parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 174
<E> fish: 36  parse_execution_context_t::eval_node_at_offset(unsigned int, block_t const*, io_chain_t const&) + 1680
<E> fish: 37  parser_t::eval_block_node(unsigned int, io_chain_t const&, block_type_t) + 242
<E> fish: 38  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t, parse_node_tree_t) + 533
<E> fish: 39  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t) + 477
<E> fish: 40  fish() [0x4721dc]
<E> fish: 41  exec_job(parser_t&, job_t*) + 6664
<E> fish: 42  parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1230
<E> fish: 43  parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 174
<E> fish: 44  parse_execution_context_t::eval_node_at_offset(unsigned int, block_t const*, io_chain_t const&) + 1680
<E> fish: 45  parser_t::eval_block_node(unsigned int, io_chain_t const&, block_type_t) + 242
<E> fish: 46  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t, parse_node_tree_t) + 533
<E> fish: 47  fish() [0x4c4e09]
<E> fish: 48  reader_read(int, io_chain_t const&) + 1130
<E> fish: 49  main + 2717
<E> fish: 50  __libc_start_main + 234
<E> fish: 51  _start + 42
Aborted (core dumped)
```

**Comment by oranja** [comment]
Hi @tastyminerals,

We're going to need more details to start to investigate your situation.
Please include:

* Your operating system (and version).
* Your fish version, and where you got it from? (fedora repository?
homebrew? built from sources?)
* Which terminal emulator you're using (and version)?
* Any other detail that you think might be relevant to this issue. What's
unique in your setup?

Thanks

**Comment by thejoeejoee** [comment]
Same problem,
```
$ lsb_release -a -u
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.3 LTS
Release:	16.04
Codename:	xenial
$ fish --version
fish, version 2.6.0
$ apt-cache policy fish
fish:
  Instalovaná verze: 2.6.0-1~xenial
  Kandidát:          2.6.0-1~xenial
  Tabulka verzí:
 *** 2.6.0-1~xenial 500
        500 http://ppa.launchpad.net/fish-shell/release-2/ubuntu xenial/main amd64 Packages
        100 /var/lib/dpkg/status
     2.2.0-3 500
        500 http://cz.archive.ubuntu.com/ubuntu xenial/universe amd64 Packages
$ bash --version
GNU bash, verze 4.3.48(1)-release (x86_64-pc-linux-gnu)
```
```
$ ./bin/install --offline
Existing installation detected at /home/thejoeejoee/.local/share/omf
Would you like to remove the existing installation? (y/N): ⏎                                                                                                                                                                                   read> Y
This will uninstall Oh My Fish and all plugins and themes from 
/home/thejoeejoee/.local/share/omf.
Are you sure you want to continue? (y/N): <E> fish: src/reader.cpp:1559: failed assertion: input_initialized
<E> fish: Backtrace:
<E> fish: 0   reader_push(wchar_t const*) + 1999
<E> fish: 1   fish() [0x437974]
<E> fish: 2   builtin_run(parser_t&, wchar_t const* const*, io_streams_t&) + 257
<E> fish: 3   exec_job(parser_t&, job_t*) + 1599
<E> fish: 4   parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1977
<E> fish: 5   parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 182
<E> fish: 6   parse_execution_context_t::eval_node_at_offset(unsigned int, block_t const*, io_chain_t const&) + 1739
<E> fish: 7   parser_t::eval_block_node(unsigned int, io_chain_t const&, block_type_t) + 234
<E> fish: 8   parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t, parse_node_tree_t) + 490
<E> fish: 9   parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t) + 429
<E> fish: 10  fish() [0x472f1f]
<E> fish: 11  exec_job(parser_t&, job_t*) + 6519
<E> fish: 12  parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1977
<E> fish: 13  parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 182
<E> fish: 14  parse_execution_context_t::eval_node_at_offset(unsigned int, block_t const*, io_chain_t const&) + 1739
<E> fish: 15  parser_t::eval_block_node(unsigned int, io_chain_t const&, block_type_t) + 234
<E> fish: 16  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t, parse_node_tree_t) + 490
<E> fish: 17  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t) + 429
<E> fish: 18  fish() [0x472f1f]
<E> fish: 19  exec_job(parser_t&, job_t*) + 6519
<E> fish: 20  parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1977
<E> fish: 21  parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 182
<E> fish: 22  parse_execution_context_t::eval_node_at_offset(unsigned int, block_t const*, io_chain_t const&) + 1739
<E> fish: 23  parser_t::eval_block_node(unsigned int, io_chain_t const&, block_type_t) + 234
<E> fish: 24  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t, parse_node_tree_t) + 490
<E> fish: 25  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t) + 429
<E> fish: 26  fish() [0x472f1f]
<E> fish: 27  exec_job(parser_t&, job_t*) + 6519
<E> fish: 28  parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1977
<E> fish: 29  parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 182
<E> fish: 30  parse_execution_context_t::run_if_statement(parse_node_t const&) + 400
<E> fish: 31  parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1275
<E> fish: 32  parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 182
<E> fish: 33  parse_execution_context_t::run_if_statement(parse_node_t const&) + 400
<E> fish: 34  parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1275
<E> fish: 35  parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 182
<E> fish: 36  parse_execution_context_t::eval_node_at_offset(unsigned int, block_t const*, io_chain_t const&) + 1739
<E> fish: 37  parser_t::eval_block_node(unsigned int, io_chain_t const&, block_type_t) + 234
<E> fish: 38  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t, parse_node_tree_t) + 490
<E> fish: 39  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t) + 429
<E> fish: 40  fish() [0x472f1f]
<E> fish: 41  exec_job(parser_t&, job_t*) + 6519
<E> fish: 42  parse_execution_context_t::run_1_job(parse_node_t const&, block_t const*) + 1977
<E> fish: 43  parse_execution_context_t::run_job_list(parse_node_t const&, block_t const*) + 182
<E> fish: 44  parse_execution_context_t::eval_node_at_offset(unsigned int, block_t const*, io_chain_t const&) + 1739
<E> fish: 45  parser_t::eval_block_node(unsigned int, io_chain_t const&, block_type_t) + 234
<E> fish: 46  parser_t::eval(std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> > const&, io_chain_t const&, block_type_t, parse_node_tree_t) + 490
<E> fish: 47  fish() [0x4bfe11]
<E> fish: 48  reader_read(int, io_chain_t const&) + 1226
<E> fish: 49  main + 2858
<E> fish: 50  __libc_start_main + 240
<E> fish: 51  _start + 41
Neúspěšně ukončen (SIGABRT) (core dumped [obraz paměti uložen])
```

**Comment by oranja** [comment]
Thanks for this extra report Josef.

So gnome-terminal, right?
Any interesting environment variable values? TERM, LOCALE, LC_*, etc..

**Comment by thejoeejoee** [comment]
Yes, gnome terminal via [Guake](https://github.com/Guake/guake) on Elementary OS Loki.
```
$ env
XDG_VTNR=7
SSH_AGENT_PID=2302
XDG_SESSION_ID=c34
XDG_GREETER_DATA_DIR=/var/lib/lightdm-data/thejoeejoee
GIO_LAUNCHED_DESKTOP_FILE_PID=2486
QT_STYLE_OVERRIDE=gtk
GPG_AGENT_INFO=/home/thejoeejoee/.gnupg/S.gpg-agent:0:1
TERM=xterm
SHELL=/bin/bash
XDG_MENU_PREFIX=gnome-
QT_LINUX_ACCESSIBILITY_ALWAYS_ON=1
GTK_MODULES=pantheon-filechooser-module:gail:atk-bridge
USER=thejoeejoee
LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.jpg=01;35:*.jpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:
QT_ACCESSIBILITY=1
DESKTOP_AUTOSTART_ID=10e36133f9f5e462fa150701710367265900000022300002
XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session0
XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0
SSH_AUTH_SOCK=/tmp/ssh-1SLij0Y1vQA6/agent.2230
SESSION_MANAGER=local/selene:@/tmp/.ICE-unix/2230,unix/selene:/tmp/.ICE-unix/2230
DEFAULTS_PATH=/usr/share/gconf/pantheon.default.path
GIO_LAUNCHED_DESKTOP_FILE=/home/thejoeejoee/.config/autostart/guake.desktop
XDG_CONFIG_DIRS=/etc/xdg/xdg-pantheon:/etc/xdg
PATH=/home/thejoeejoee/bin:/home/thejoeejoee/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games
DESKTOP_SESSION=pantheon
QT_IM_MODULE=ibus
XDG_SESSION_TYPE=x11
PWD=/home/thejoeejoee/projects/oh-my-fish
XMODIFIERS=@im=ibus
LANG=cs_CZ.UTF-8
GDM_LANG=cs
MANDATORY_PATH=/usr/share/gconf/pantheon.mandatory.path
GDMSESSION=pantheon
XDG_SEAT=seat0
HOME=/home/thejoeejoee
SHLVL=1
LANGUAGE=cs
GNOME_DESKTOP_SESSION_ID=this-is-deprecated
XDG_SESSION_DESKTOP=pantheon
LOGNAME=thejoeejoee
XDG_DATA_DIRS=/usr/share/pantheon:/usr/local/share/:/usr/share/
DBUS_SESSION_BUS_ADDRESS=unix:abstract=/tmp/dbus-5i8vzgR4SS,guid=14998e792152c0caff59685a59d3418f
LESSOPEN=| /usr/bin/lesspipe %s
GTK_CSD=1
XDG_RUNTIME_DIR=/run/user/1000
DISPLAY=:0
XDG_CURRENT_DESKTOP=Pantheon
LESSCLOSE=/usr/bin/lesspipe %s %s
XAUTHORITY=/home/thejoeejoee/.Xauthority
_=/usr/bin/env
```

**Comment by tastyminerals** [comment]
* Your operating system (and version).
4.13.4-1-MANJARO

* Your fish version, and where you got it from? (fedora repository?
homebrew? built from sources?)
fish, version 2.6.0
installed via pacman package manager

* Which terminal emulator you're using (and version)?
xfce4-terminal

```bash
echo $TERM 
xterm-256color
```
* Any other detail that you think might be relevant to this issue. What's
unique in your setup?

**.bashrc**
```bash
#
# ~/.bashrc
#

[[ $- != *i* ]] && return

colors() {
	local fgc bgc vals seq0

	printf "Color escapes are %s\n" '\e[${value};...;${value}m'
	printf "Values 30..37 are \e[33mforeground colors\e[m\n"
	printf "Values 40..47 are \e[43mbackground colors\e[m\n"
	printf "Value  1 gives a  \e[1mbold-faced look\e[m\n\n"

	# foreground colors
	for fgc in {30..37}; do
		# background colors
		for bgc in {40..47}; do
			fgc=${fgc#37} # white
			bgc=${bgc#40} # black

			vals="${fgc:+$fgc;}${bgc}"
			vals=${vals%%;}

			seq0="${vals:+\e[${vals}m}"
			printf "  %-9s" "${seq0:-(default)}"
			printf " ${seq0}TEXT\e[m"
			printf " \e[${vals:+${vals+$vals;}}1mBOLD\e[m"
		done
		echo; echo
	done
}

[[ -f ~/.extend.bashrc ]] && . ~/.extend.bashrc

[ -r /usr/share/bash-completion/bash_completion   ] && . /usr/share/bash-completion/bash_completion

# aliases
alias ls="ls --color=auto --group-directories-first"
alias docker_rm_containers="docker rm $(docker ps -a -q) -f"
alias docker_rm_images="docker rmi $(docker images -q) -f"

# exports 
export EDITOR=vim
export HISTCONTROL=ignoreboth:erasedups
# torch init
#. /home/pavel/torch/install/bin/torch-activate
# Add RVM to PATH for scripting. Make sure this is the last PATH variable change.
export PATH="$PATH:$HOME/.rvm/bin"
# virtualenvwrapper
export WORKON_HOME="$HOME/.virtualenvs"
VIRTUALENVWRAPPER_PYTHON='/usr/bin/python3' # This needs to be placed before the virtualenvwrapper command
source /usr/local/bin/virtualenvwrapper.sh
export _JAVA_OPTIONS='-Dawt.useSystemAAFontSettings=on -Dswing.aatext=true -Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel'
```

**.extend.bashrc**
```bash
#
# ~/.extend.bashrc
#

# Change the window title of X terminals
case ${TERM} in
	xterm*|rxvt*|Eterm*|aterm|kterm|gnome*|interix|konsole*)
		PROMPT_COMMAND='echo -ne "\033]0;${USER}@${HOSTNAME%%.*}:${PWD/#$HOME/\~}\007"'
		;;
	screen*)
		PROMPT_COMMAND='echo -ne "\033_${USER}@${HOSTNAME%%.*}:${PWD/#$HOME/\~}\033\\"'
		;;
esac

use_color=true

# Set colorful PS1 only on colorful terminals.
# dircolors --print-database uses its own built-in database
# instead of using /etc/DIR_COLORS.  Try to use the external file
# first to take advantage of user additions.  Use internal bash
# globbing instead of external grep binary.
safe_term=${TERM//[^[:alnum:]]/?}   # sanitize TERM
match_lhs=""
[[ -f ~/.dir_colors   ]] && match_lhs="${match_lhs}$(<~/.dir_colors)"
[[ -f /etc/DIR_COLORS ]] && match_lhs="${match_lhs}$(</etc/DIR_COLORS)"
[[ -z ${match_lhs}    ]] \
	&& type -P dircolors >/dev/null \
	&& match_lhs=$(dircolors --print-database)
[[ $'\n'${match_lhs} == *$'\n'"TERM "${safe_term}* ]] && use_color=true

if ${use_color} ; then
	# Enable colors for ls, etc.  Prefer ~/.dir_colors #64489
	if type -P dircolors >/dev/null ; then
		if [[ -f ~/.dir_colors ]] ; then
			eval $(dircolors -b ~/.dir_colors)
		elif [[ -f /etc/DIR_COLORS ]] ; then
			eval $(dircolors -b /etc/DIR_COLORS)
		fi
	fi

	if [[ ${EUID} == 0 ]] ; then
		PS1='\[\033[01;31m\][\h\[\033[01;36m\] $PWD\[\033[01;31m\]]\$\[\033[00m\] '
	else

        #PS1="\n\[\e[0;34m\]┌─[\[\e[1;36m\u\e[0;34m\]]──[\e[1;37m\w\e[0;34m]──[\[\e[1;36m\]${HOSTNAME%%.*}\[\e[0;34m\]]\[\e[1;35m\]: \$\[\e[0;34m\]\n\[\e[0;34m\]└────╼ _\[\e[1;35m\]\[\e[00;00m\]"
		PS1='\[\033[01;32m\][\u@\h\[\033[01;37m\] $PWD\[\033[01;32m\]]\$\[\033[00m\] '
	fi

	alias ls='ls --color=auto'
	alias grep='grep --colour=auto'
	alias egrep='egrep --colour=auto'
	alias fgrep='fgrep --colour=auto'
else
	if [[ ${EUID} == 0 ]] ; then
		# show root@ when we don't have colors
		PS1='\u@\h \W \$ '
	else
		PS1='\u@\h \w \$ '
	fi
fi

unset use_color safe_term match_lhs sh

alias cp="cp -i"                          # confirm before overwriting something
alias df='df -h'                          # human-readable sizes
alias free='free -m'                      # show sizes in MB
alias np='nano -w PKGBUILD'
alias more=less

xhost +local:root > /dev/null 2>&1

complete -cf sudo

# Bash won't get SIGWINCH if another process is in the foreground.
# Enable checkwinsize so that bash will check the terminal size when
# it regains control.  #65623
# http://cnswww.cns.cwru.edu/~chet/bash/FAQ (E11)
shopt -s checkwinsize

shopt -s expand_aliases


# export QT_SELECT=4

# Enable history appending instead of overwriting.  #139609
shopt -s histappend

#
# # ex - archive extractor
# # usage: ex <file>
ex ()
{
  if [ -f $1 ] ; then
    case $1 in
      *.tar.bz2)   tar xjf $1   ;;
      *.tar.gz)    tar xzf $1   ;;
      *.bz2)       bunzip2 $1   ;;
      *.rar)       unrar x $1     ;;
      *.gz)        gunzip $1    ;;
      *.tar)       tar xf $1    ;;
      *.tbz2)      tar xjf $1   ;;
      *.tgz)       tar xzf $1   ;;
      *.zip)       unzip $1     ;;
      *.Z)         uncompress $1;;
      *.7z)        7z x $1      ;;
      *)           echo "'$1' cannot be extracted via ex()" ;;
    esac
  else
    echo "'$1' is not a valid file"
  fi
}

# better yaourt colors
export YAOURT_COLORS="nb=1:pkg=1:ver=1;32:lver=1;45:installed=1;42:grp=1;34:od=1;41;5:votes=1;44:dsc=0:other=1;35"
```

**Comment by oranja** [comment]
This issue is directly related to https://github.com/fish-shell/fish-shell/issues/4206.
It should be fixed in fish 2.7.0, but we will still try to see if we can work around it for 2.6.0 users.

**Comment by sagebind** [comment]
Closing, since this is a Fish bug and there's not a whole lot we can do to work around it without changing the installer behavior.
